client/src/components/Donation/DonateForm.tsx

```diff
@@ -72,8 +102,12 @@ const mapDispatchToProps = {
   updateDonationFormState
 };
 
-class DonateForm extends Component {
-  constructor(...args) {
+class DonateForm extends Component<DonateFormProps, DonateFormState> {
+  static displayName = 'DonateForm';
+  durations: { month: string; onetime: string };
+  amounts: { month: number[]; onetime: number[] };
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  constructor(...args: any) {
     super(...args);
```

Comment:
```suggestion
  constructor(props: any) {
    super(props);
```
I don't entirely know how this was working with the spread operator, but it shouldn't have been there.